draft-webtransport-http2.md

```diff
@@ -5,7 +5,7 @@ docname: draft-webtransport-http2-latest
 
```

Comment:
Line 385: "subsequent..."  - What happens to open WT Streams?

Reply:
Needs an issue, we can either let them wrap up or close immediately, tempted to say the latter